3rdParty/dpp/nlohmann/json_fwd.hpp

```diff
@@ -0,0 +1,64 @@
+#ifndef INCLUDE_NLOHMANN_JSON_FWD_HPP_
+#define INCLUDE_NLOHMANN_JSON_FWD_HPP_
+
+#include <cstdint> // int64_t, uint64_t
+#include <map> // map
+#include <memory> // allocator
+#include <string> // string
+#include <vector> // vector
+
+/*!
+@brief namespace for Niels Lohmann
+@see https://github.com/nlohmann
+@since version 1.0.0
+*/
+namespace nlohmann
```

Comment:
No namespace conflict or ODR violation with `3rdParty/nlohmann/json.hpp`?

Reply:
Not as far as I can tell. Not getting any compiler warnings, nlohmann seems to be accessible and usable just fine.

Reply:
The ODR part scares me a bit. Since the linker can mix things up across the two files. We're probably getting lucky because they're only ever being used in two completely separate contexts.

I'll merge, then change the namespace of our existing nlohmann as the short-term fix. The proper solution is to point both CC and DPP to the same nlohmann file. But that could be a bit messy.